@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -1197,185 +1182,6 @@ export class TileGeometryCreator {
         }
     }
 
-    /**
```

Comment:
These two functions below getRenderStyle / getLayoutStyle were duplicated in PoiManager. I've moved them to TextStyleCache.